src/client/Microsoft.Identity.Client/Core/Constants.cs

```diff
@@ -13,7 +13,7 @@ internal static class Constants
         public const int CodeVerifierByteSize = 32;
 
         public const string UapWEBRedirectUri = "https://sso"; // only ADAL supports WEB
-        public const string DefaultRedirectUri = "urn:ietf:wg:oauth:2.0:oob";
+        public const string DefaultRedirectUri = "msal4a1aa1d5-c567-49d0-ad0b-cd957a47f842://auth"; // in msidentity-samples-testing tenant -> PublicClientSample
```

Comment:
Is this redirect URI a default one for any applications?

Reply:
good point. have fixed.

---
In reply to: [353460272](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1531#discussion_r353460272) [](ancestors = 353460272)